backend/mongodb.js

```diff
@@ -17,7 +16,7 @@ mongoose.connect(url).then(() => {
   console.log("You have an error: " + error);
 });
 
-const schema = new mongoose.Schema({
+const Movieschema = new mongoose.Schema({
```

Comment:
```suggestion
const Moviechema = new mongoose.Schema({
```

the schema is of one entity dont need to have its name plural